challenge.js

```diff
@@ -6,9 +6,8 @@
  *		to a web page without refreshing it?
  *
  *	Answer below.
- */
 
-// ANSWER
+ */ Through the console
```

Comment:
That's one option. You can also do it in one of the page scripts if it's your site. That's step one. What's next? 1/2